### PR TITLE
Added items

### PR DESCRIPTION
**Pull request by creatorfromhell**

### CODE REVIEW

File: TrueCraft.Core/Logic/Items/BookItem.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+namespace TrueCraft.Core.Logic.Items
+{
+    public class BookItem : ItemProvider
+    {
+        public static readonly short ItemID = 0x154;
+
+        public override short ID { get { return 0x154; } }
+
+        public override sbyte MaximumStack { get { return 64; } }
```
Comment (SirCmpwn) [inline TrueCraft.Core/Logic/Items/BookItem.cs]: Don't override this unless it's unusual. The default is 64 and you can omit this if the item in question can be stacked to 64.

File: TrueCraft.Core/Logic/Items/BucketItem.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        public static readonly short ItemID = 0x145;
+
+        public override short ID { get { return 0x145; } }
+
+        public override sbyte MaximumStack { get { return 1; } }
+
+        public override string DisplayName { get { return "Bucket"; } }
+    }
+
+    public class LavabucketItem : BucketItem
```
Comment (SirCmpwn) [inline TrueCraft.Core/Logic/Items/BucketItem.cs]: Case error - `LavaBucketItem`

File: TrueCraft.Core/Logic/Items/FlintandSteelItem.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+using System;
+using TrueCraft.API.Logic;
+
+namespace TrueCraft.Core.Logic.Items
+{
+    public class FlintandSteelItem : ItemProvider
```
Comment (SirCmpwn) [inline TrueCraft.Core/Logic/Items/FlintandSteelItem.cs]: Case error - `FlintAndSteelItem`

File: TrueCraft.Core/Logic/Items/GoldenBootsItem.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+{
+    public class GoldenBootsItem : ArmourItem
+    {
+        public static readonly short ItemID = 0x13D;
+
+        public override short ID { get { return 0x13D; } }
+
+        public override ArmourMaterial Material { get { return ArmourMaterial.Gold; } }
+
+        public override string DisplayName { get { return "Golden boots"; } }
```
Comment (SirCmpwn) [inline TrueCraft.Core/Logic/Items/GoldenBootsItem.cs]: Case error - `Golden Boots`

File: TrueCraft.Core/Logic/Items/ShearsItem.cs [hunk 5]
```diff
@@ -0,0 +1,16 @@
+using System;
+using TrueCraft.API.Logic;
+
+namespace TrueCraft.Core.Logic.Items
+{
+    public class ShearsItem : ItemProvider
```
Comment (SirCmpwn) [inline TrueCraft.Core/Logic/Items/ShearsItem.cs]: Shears is a tool.

**Comment by SirCmpwn** [comment]
Overall this looks great! Thanks for taking the time to put this together. A couple of other tweaks:

* Can you make iron and wooden doors share a base class?
* Can you make all weapons of a given type share a base class? For example, IronSwordItem and WoodenSwordItem can both inherit from SwordItem.

**Comment by creatorfromhell** [comment]
Updated with fixes.

**Comment by SirCmpwn** [comment]
This has been merged, but GitHub didn't pick up on it. Thanks!

**Pull request closed without merging**